lib/index.ts

```diff
@@ -194,11 +198,31 @@ export class Compiler {
   private _compileParenthesizedTypeNode(node: ts.ParenthesizedTypeNode): string {
     return this.compileNode(node.type);
   }
+  private _compileImportDeclaration(node: ts.ImportDeclaration): string {
+    if (this.options.traverseImports) {
+      const importedSym = this.checker.getSymbolAtLocation(node.moduleSpecifier);
+      if (importedSym && importedSym.declarations) {
+        // this._compileSourceFile will get called on every imported file when traversing imports. 
+        // it's important to check that _compileSourceFile is being run against the topNode 
+        // before adding the file wrapper for this reason.
+        return importedSym.declarations.map(declaration => this.compileNode(declaration)).join("");
```

Comment:
@dsagal I updated the comment here - also cleaned up the returns for simplicity (there were two `return ''` before)